meilisearch-http/src/option.rs

```diff
@@ -51,6 +52,7 @@ const MEILI_LOG_LEVEL: &str = "MEILI_LOG_LEVEL";
 #[cfg(feature = "metrics")]
 const MEILI_ENABLE_METRICS_ROUTE: &str = "MEILI_ENABLE_METRICS_ROUTE";
 
+const DEFAULT_CONFIG_FILE_PATH: &str = "./config.yml";
```

Comment:
Just found this typo 🥲 I will try to add some test cast against such cases.
```suggestion
const DEFAULT_CONFIG_FILE_PATH: &str = "./config.toml";
```

Reply:
Fixed in 91accc0